docker-compose/kitchen-sink/docker-compose.yml

```diff
@@ -377,6 +377,9 @@ services:
       zookeeperServers: zk1:2181,zk2:2181,zk3:2181
       configurationStoreServers: zk1:2181,zk2:2181,zk3:2181
       pulsar.zookeeper-uri: zk1:2181,zk2:2181,zk3:2181
+      SQL_PREFIX_pulsar.broker-service-url: http://proxy1:8080
```

Comment:
The configuration `broker-service-url` is deprecated, `web-service-url` will instead of it.

Reply:
But the setting still exists in the image.  Either it needs to be removed completely, or it needs to be updated to be valid in the given environment, no?  I have assumed it's left in there because it may still be used by something which is why it wasn't removed when it was replaced with the web- alternative.
I'm happy to go either way - remove broker- completely, or keep it and make it correct for this environment.  It doesn't seem to make any sense to leave it and have it point to an invalid URL.  Which way should I go?

Reply:
@gaoran10 Any feedback on this?

Reply:
Sorry late response, I think the config `web-service-url` could replace the config `broker-service-url` completly, if the config `web-service-url` is set, the `broker-service-url` will be useless, refer to https://github.com/apache/pulsar/blob/master/pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarConnectorConfig.java#L90